1.10.5/buster/Dockerfile

```
@@ -128,7 +128,7 @@ RUN pip install "${AIRFLOW_MODULE}" \
 RUN cd usr/local/lib/python${PYTHON_MAJOR_MINOR_VERSION}/site-packages/airflow/www_rbac \
   && npm install \
   && npm run prod \
-  && rm -rf node_modules
+  && rm -rf node_modules package.json yarn.lock packages.lock
```

We should remove `package-lock.json` too
```suggestion
  && rm -rf node_modules package.json yarn.lock packages.lock package-lock.json
```

---

https://app.circleci.com/pipelines/github/astronomer/ap-airflow/1434/workflows/2e5ad218-b0f3-4c2f-86c0-399c99f20838/jobs/35694

---

`packages.lock` is not a real file -- `package-lock.json` is what I _meant_ to put.